pyintegration/deephaven2/__init__.py

```diff
@@ -5,17 +5,21 @@
 unlocks the unique and tremendous power of Deephaven to the Python community.
```

Comment:
To me this sounds like too much of a blatant marketing plug to belong in the source code.